deeplake/integrations/pytorch/common.py

```diff
@@ -30,10 +30,15 @@ def collate_fn(batch):
     elif isinstance(elem, list) and all(
         map(lambda e: isinstance(e, np.ndarray), elem)
     ):  # special case for video query api
-        if len(elem[0].shape) > 1 and elem[0].shape[1] not in [
-            2,
-            3,
-        ]:  # checking whether it is not a polygon
+        if (
+            len(elem) > 0
+            and len(elem[0].shape) > 1
+            and elem[0].shape[1]
+            not in [
```

Comment:
unrelated but shouldn't we look at more than just the shape to know whether it is a polygon or not?